packages/account/src/providers/assets/utils/network.test.ts

```diff
@@ -0,0 +1,69 @@
+import { Asset } from '../types'
+import { getAssetEth, getAssetFuel, getAssetNetwork, getAssetWithNetwork, getDefaultChainId } from '../utils/network';
+import { assets } from '../index'
+import { CHAIN_IDS } from '../../chains'
+
+/**
+ * @group node
+ */
+describe('Network Utils', () => {
+  test('getDefaultChainId', async () => {
+    expect(getDefaultChainId('ethereum')).toBe(11155111);
+    expect(getDefaultChainId('fuel')).toBe(0);
+  })
+
+  test('getAssetWithNetwork - Ethereum', async () => {
```

Comment:
```suggestion
test('getAssetNetwork - Ethereum', async () => {
```

Reply:
Wait, all my suggestions were based on you using `getAssetNetwork` but I just realized that `getAssetWithNetwork` also exists... Which one should  these tests be testing? :D

Reply:
Ah my bad there was a mixup. Fixed now lol